test/external/external_model_benchmark.py

```diff
@@ -103,29 +98,44 @@ def benchmark_model(m, devices, validate_outs=False):
     del ort_sess
 
   if validate_outs:
-    rtol, atol = 2e-3, 2e-3  # tolerance for fp16 models
-    inputs = {k:Tensor(inp) for k,inp in np_inputs.items()}
-    tinygrad_model = get_run_onnx(onnx_model)
-    tinygrad_out = tinygrad_model(inputs)
+    if any(t == np.dtype(np.float16) for t in input_types.values()): rtol, atol = 3e-3, 1e-3  # tolerance for fp16 models
```

Comment:
raised rtol for `half` ... 
otherwise I would have to set `half_to_float_to_half` to non normalization ops.
I tried a lot of `float()` then `cast(half)` casting to find where the inaccuracy was originating from and it was pretty difficult.
I ended up just wrapping the normalization with cast to copy how cifar casts.

Reply:
this is fine it think